experiments/dataSources/openAI/draftAnswers/Readme.MD

```diff
@@ -0,0 +1,114 @@
+# Ask ChatGPT to draft answers in your forms. 
+ 
+ This sample page shows how the ChatGPT API can help draft field values for any data you manage with Skuid. The application uses a Skuid action sequence that uses information from a record to create a prompt that is sent to ChatGPT.  The user can review the resulting response and iterate on the responses provided by the AI Engine. Finally the output  is copied into the specified field in your application.  The generic nature of the application means it can be bound to any field in your application and will return the output to the correct place. 
+
+## Instructions
+- Page API:  V2
+- Data source: REST data source named "OpenAi" connecting to your OpenAI instance per [instruction](openAI)
+- Design system: None 
+- Page XML:  [Copy the XML from this page](UpdateAnyField_w_ChatGPT.xml?raw=true), or save it as an XML file, and upload it as a new page in Skuid's Pages.
+
+## Notes
+
+### Models:  
+
+The application uses 2 sets of models. 
+
+The first three models represent Business Object you might find in your CRM.  For the sake of this demo they are UI Only models,  but you might be creating new Salesforce records, or new survey questions in a Postgres table etc.  
+
+The second set of models are used for the Chat function. 
+
+1.  UI Only “Chat Thread Display” model. 
+This model will show the output from the ChatGPT interaction and hold the full thread of the discussion. 
+
+2. REST model for interacting with ChatGPT. 
+This model will handle the communication with the Chat Engine. 
+
+<img src="Chat_Completion_Model.png" width="500"></img>
+
+
+### Snippets: 
+Two snippets are required for interacting with this data source. 
+
+1. Request body construction snippet. 
+
+This snippet creates the request body in the format necessary for submitting to the OpenAI API. 
+
+- Resource type:  Generic JS Snippet
+- Snippet name:  requestBody
+Code: 
```

Comment:
While I like having the code here for easy reference—we don't do that in the other instructions. It can also mean making updates in one area could cause a mismatch between the XML and the reference. 

I'd consider adding the source code to the other instructions or removing it from here (assuming the XML snippet is as well-commented as this reference)